src/ATen/function_wrapper.py

```diff
@@ -305,10 +327,43 @@ def process_option(option, output_options):
         # another function-only variant can exist without the name colliding
         option['method_prefix'] = 'm_' if is_method and not is_function else ''
         env = nested_dict(option, top_env)
-        top_env['type_method_declarations'].append(
-            TYPE_METHOD_DECLARATION.substitute(env))
-        top_env['type_method_definitions'].append(
-            TYPE_METHOD_DEFINITION.substitute(env))
+
+        broadcast_arg = get_broadcast_argument(option)
+        if broadcast_arg is None:
+            option['method_prefix_derived'] = option['method_prefix']
+            top_env['type_method_declarations'].append(
+                TYPE_METHOD_DECLARATION.substitute(env))
+            top_env['type_method_definitions'].append(
+                TYPE_METHOD_DEFINITION.substitute(env))
+        else:
+            option['method_prefix_derived'] = 's_' + option['method_prefix']
```

Comment:
Make a comment about what `s_` means in this context

Reply:
done.